validate-pr/dangerfile.js

```diff
@@ -16,28 +18,28 @@ if (_.get(pr, 'user.type') === 'Bot') {
 
 const title = _.trim(_.get(pr, 'title'))
 let issueType;
-let passed;
-
-if (title.includes('PS-')) {
-    issueType = 'PS';
-    passed = true;
-} else if (title.includes('INSIGHT-')) {
-    issueType = 'INSIGHT';
-    passed = true;
-} else {
-    passed = false;
-    fail('PR Validation Failed :disappointed:');
+
+try {
+    [, issueType] = title.match(/^(\w+)-.*$/);
+} catch {
+    fail('PR Validation Failed :disappointed:.  Could not detect issue type from PR title.');
+    process.exit(0);
```

Comment:
Simpler to just exit immediately than keep the `passed` flag around and check for it at the end.  Calling `fail()` will make the build fail, even if we exit the process with no error code.